src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -394,13 +394,16 @@ impl ContainerModel {
         self.set_visibility(!self.is_active())
     }
 
-    fn set_visualization(&self, visualization:Option<visualization::Instance>) {
-        if let Some(visualization) = visualization {
-            let size = self.size.get();
-            visualization.set_size.emit(size);
-            self.view.add_child(&visualization);
-            self.visualization.replace(Some(visualization));
+    fn set_visualization
+    (&self, visualization:visualization::Instance, preprocessor:&frp::Any<EnsoCode>) {
```

Comment:
couldyou change the types to be `language::enso::Code` (of course `language` part can be skipped in imports), as we've been talking on DM, please?